app/views/LibraryInfoPage.scala.html

```diff
@@ -51,7 +51,7 @@ <h1>
   </header>
 
   @partials.leak_detection_banner(linkToLeakDetection)
-  @defining(repositoryModules.fold(Seq.empty[Dependency])(_.allDependencies)){ dependencies =>
+  @defining(metaInfo.headOption.fold(Seq.empty[Dependency])(_.allDependencies)){ dependencies =>
```

Comment:
can we use `@defining(metaInfo.headOption){ latestVersion =>` to explain what's going on, and avoid multiple calls to `metaInfo.headOption`?